style.css

```diff
@@ -709,3 +709,282 @@ small {
+}
+.breadcrumbs li:first-child::after {
+    display: none;
+}
+.breadcrumbs a {
+    color: #F5F8FB;
+    text-decoration: none;
+    transition: opacity 0.2s linear;
+}
+.breadcrumbs a:hover {
+    opacity: 0.8;
+}
+.wrapper {
+    display: flex;
+    margin-inline: auto;
+    padding: 40px 15px 60px;
+    max-width: 1310px;
+}
+.sidebar {
+    flex-shrink: 0;
+    flex-grow: 0;
+    flex-basis: 328px;
+    margin-right: 36px;
+}
+.sidebar-box {
+    background-color: #1D1C28;
+    margin-bottom: 16px;
+    padding: 20px 15px;
+}
+.sidebar__title {
+    display: block;
+    font: 24px/32px var(--font-secondary);
+    margin-bottom: 16px;
+    position: relative;
+    padding-bottom: 16px;
+}
+.sidebar__title::after {
+    content: "";
+    position: absolute;
+    z-index: 2;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    height: 2px;
+    background-color: rgba(191, 197, 238, 0.12);
+}
+.sidebar__list,
+.tags {
+    list-style: none;
+    margin: 0;
+    padding: 0 10px;
+    font-size: 14px;
+    line-height: 20px;
+    font-weight: 600;
+}
+.sidebar__item {
+    padding-top: 20px;
+}
+.sidebar__item:first-child {
+    padding-top: 0;
+}
+.sidebar__link,
+.tags__link {
+    text-decoration: none;
+    color: #F5F8FB;
+}
+.tags {
+    display: flex;
+    flex-wrap: wrap;
+    gap: 8px;
+}
+.tags__item {
+    border: 1px solid rgba(208, 215, 251, 0.17);
+    padding: 0 8px;
+}
+.post {
+    flex-shrink: 1;
+    flex-grow: 1;
+    flex-basis: auto;
+    font-size: 18px;
+    font-weight: 400;
+    color: #F8F7FF;
+}
+figure {
+    position: relative;
+    margin: 0 0 20px;
+}
+figcaption {
+    position: absolute;
+    z-index: 2;
+    bottom: 15px;
+    left: 25px;
+    font-size: 16px;
+    line-height: 24px;
+    font-weight: 500;
+    color: #FFF;
+}
+.content__title {
+    font: 36px/48px var(--font-secondary);
+    margin: 0 0 20px;
+}
+.post__btn {
+    margin-bottom: 32px;
+}
+.blog__info {
+    color: rgba(239, 245, 250, 0.82);
+}
+.post p {
+    margin: 0 0 16px;
+}
+.post ul {
+    margin: 0 0 16px;
+    padding-left: 16px;
+}
+.post li {
+    padding-top: 10px;
+}
+.post li:first-child {
+    padding-top: 0;
+}
+.post a {
+    text-decoration: none;
+    color: #78BBFA;
+    transition: opacity 0.2s linear;
+}
+.post a:hover {
+    opacity: 0.8;
+}
+.post .pointer-list {
+    padding: 0;
+    list-style: none;
+}
+.pointer-list li {
+    position: relative;
+    padding: 0 0 0 40px;
+    margin-top: 10px;
+}
+.pointer-list li:first-child {
+    margin-top: 0;
+}
+.pointer-list li::after {
+    content: '';
+    position: absolute;
+    top: 2px;
+    left: 0;
+    z-index: 2;
+    width: 22px;
+    height: 25px;
+    background-image: url(img/icons-logo.svg);
+}
+.blockquote {
+    background-color: #002754;
+    border-radius: 16px;
+    font-size: 24px;
+    line-height: 30px;
+    font-weight: 600;
+    text-transform: capitalize;
+    padding: 16px;
+    margin-bottom: 40px;
+}
+blockquote {
+    margin: 0 0 8px;
+}
```

Comment:
How about to concat these styles and above?

Reply:
Done